templates/prelogin/base.html

```diff
@@ -205,6 +205,32 @@ <h3 class="modal-title">
                 </div>
             </div>
         </div>
+        <div id="contactUsFormErrors"
+             class="modal fade">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">{% trans 'Close' %}</span></button>
+                        <h3 class="modal-title">
+                            {% trans "Ooops!" %}
+                        </h3>
+                    </div>
+                    <div class="modal-body">
+                        <p>
+                            {% comment %}
+                                There is only one field that can result in errors,
+                                so it's safe to assume the problem is the e-mail
+                                address.
```

Comment:
what is up with this error message? If a user actually sees it, this wouldn't make much sense to them. @kaapstorm feel free to make those changes re: error dict.